package.json

```diff
@@ -0,0 +1,49 @@
+{
+  "name": "aqovia-vscode-extension-pack",
+  "displayName": "Aqovia VSCode ExtensionPack",
+  "description": "Aqovia Visual Studio Code Extension Pack",
+  "version": "1.3.34",
+  "publisher": "Aqovia",
+  "license": "SEE LICENSE IN LICENSE.txt",
+  "engines": {
+    "vscode": "^1.80.0"
+  },
+  "scripts": {
+    "pack": "vsce package --out VSCodeExtensionsPack.vsix"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/Aqovia/VisualStudioCode.ExtensionPack"
+  },
+  "categories": [
+    "Extension Packs"
+  ],
+  "extensionPack": [
+    "hnw.vscode-auto-open-markdown-preview",
+    "redhat.vscode-yaml",
+    "Zignd.html-css-class-completion",
+    "streetsidesoftware.code-spell-checker",
+    "ms-dotnettools.csharp",
+    "donjayamanne.git-extension-pack",
+    "eriklynd.json-tools",
+    "dbaeumer.vscode-eslint",
+    "mrmlnc.vscode-less",
+    "DavidAnson.vscode-markdownlint",
+    "ms-mssql.mssql",
+    "leizongmin.node-module-intellisense",
+    "christian-kohler.npm-intellisense",
+    "polymer.polymer-ide",
+    "ms-vscode.PowerShell",
+    "ms-python.python",
+    "pflannery.vscode-versionlens",
+    "DotJoshJohnson.xml",
```

Comment:
I don't think this is actively maintained any more. We may be better off installing `redhat.vscode-xml` but I am unsure of feature-parity.

Reply:
They seem similar, but I think you'd have to use both for a while to really get a feel for the upsides and downsides. I'll replace it and see if anyone has issues.